wire-ios-data-model/Tests/UseCases/HasSelfUserValidE2EICertificatesForAllClientsUseCaseTests.swift

```diff
@@ -53,7 +53,7 @@ final class IsSelfUserE2EICertifiedUseCaseTests: ZMBaseManagedObjectTest {
         // Given
         mockSafeCoreCrypto.coreCrypto.getUserIdentitiesConversationIdUserIds_MockMethod = { conversationID, userIDs in
             XCTAssertEqual(conversationID, .init(base64Encoded: "qE4EdglNFI53Cm4soIFZ/rUMVL4JfCgcE4eo86QVxSc=")!)
-            XCTAssertEqual(userIDs, ["36dfe52f-157d-452b-a9c1-98f7d9c1815d@example.com"])
+            XCTAssertEqual(userIDs, ["36dfe52f-157d-452b-a9c1-98f7d9c1815d"]) // TODO []: after CC update it should have the prefix "@example.com"
```

Comment:
Can you please create a ticket for this TODO and add it here? e.g [WPB-680]

Reply:
@agisilaos  We don't have ticket and don't know when it will be fixed on CC side

Reply:
Sorry, I forget the ticket number, fixed.